modules/core/src/v2/wallet.ts

```diff
@@ -1262,7 +1262,9 @@ export class Wallet {
    * @param {Number} params.forwarderVersion The version of address to create, if applicable
    * @param {Boolean} params.lowPriority Ethereum-specific param to create address using low priority fee address
    * @param {String} params.baseAddress base address of the wallet(optional parameter)
-   * @param {Boolean} params.strictMode set to strictly verify the created address
+   * @param {Boolean} params.strictMode set to strictly verify the created address.Throws error if address verification is skipped for any reason.
```

Comment:
how about we call this `allowSkipValidation` instead? Default is `true` and we allow setting it to `false`

Reply:
Done.